### DISCUSSION TITLE
第4节课笔记(23班)

### DISCUSSION THREAD

**Discussion by wangyijunlyy** [InternLM/Tutorial#418] [笔记]
笔记提交

**Comment by i-square** [comment]
https://i-square.github.io/p/InternLM-tutorial-campsection4-XTuner-based-LLM-Single-GPU-Low-cost-Finetune-Practice/

**Comment by yaoyaotaba** [comment]
https://zhuanlan.zhihu.com/p/678917540

**Comment by PlateAss** [comment]
https://plateass.github.io/class4

**Comment by YangZhC** [comment]
https://blog.csdn.net/donotpython/article/details/135764909

**Comment by Epi10gue** [comment]
http://t.csdnimg.cn/UQGDy
